raiden/network/proxies/token_network.py

```diff
@@ -916,7 +904,7 @@ def settle(
                 partner_locked_amount,
                 partner_locksroot,
             ) is False:
-                raise ChannelIncorrectStateError('local state can not be used to call settle')
+                raise RaidenUnrecoverableError('local state can not be used to call settle')
```

Comment:
It verifies we can compute the same balance hashes with the ones we get from the contract. Since it's in verification stage, i think you're right and we should raise a recoverable error